Libraries/LibWeb/CSS/Default.css

```diff
@@ -95,7 +87,7 @@ input[type=range]::-webkit-slider-runnable-track {
     height: 4px;
     width: 100%;
     margin-top: 6px;
-    background-color: Background;
+    background-color: AccentColorText;
```

Comment:
Should this be `AccentColor`? Using the text colour for non-text feels odd. Same for `progress::-webkit-progress-bar` below.

Reply:
I'm using `AccentColorText` here because I can rely on it having enough contrast with the `AccentColor` progress bar.

Reply:
Ah gotcha, I missed that.